woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -206,6 +207,54 @@ def test_accessor_getattr(sample_df):
         sample_df.ww.not_present
 
 
+def test_getitem(sample_df):
```

Comment:
One thing I notice in looking at this test again, is that we are not verifying that any user types set during init are carried through to the final subset dataframe. We can modify this test slightly to confirm this happens. My suggestion would be set the logical type of Age to `Double` on init and also add a custom semantic tag, something like `custom_tag`, and then update the two tests that include age to check that these values are set correctly on the subset.

Reply:
✅  7ac4fa4